sig-autoscaling/charter.md

```diff
@@ -0,0 +1,74 @@
+# SIG Autoscaling Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+Covers development and maintenance of componets for automated scaling in
+Kubernetes.  This includes automated vertical and horizontal pod
+autoscaling, initial resource estimation, cluster-proportional system
+component autoscaling, and autoscaling of Kubernetes clusters themselves.
```

Comment:
Currently cluster proportional autoscaling is not owned by sig-autoscaling:
* addon-resizer is maintained by sig-instrumentation (AFAIK it's primarily used for heapster and metrics-server)
* kube-dns-autoscaler is maintained by sig-networking

Ideally sig-autoscaling should take ownership of this use-case, but we're not there yet.

Reply:
oh, do we not own the addon-resizer?  I thought it was under our purview from the pre-sig-instrumentation Heapster days, and I didn't think it got transferred over, since it's an autoscaler.  

We probably *should* own those, at any rate. @kubernetes/sig-instrumentation-misc

Reply:
> addon-resizer is maintained by sig-instrumentation (AFAIK it's primarily used for heapster and metrics-server)

I was not aware of this, and it's also not in our sig components. I was under the impression that sig-autoscaling own it.

Reply:
I mean no-one is really doing much work on addon-resizer, but I believe @kawych and @x13n are the ones maintaining it.

Reply:
Every kubernetes piece of code needs to be owned by a SIG.  I think it makes sense for us to own it, especially since it doesn't fall under the responsibilities of SIG instrumentation normally.  @kawych and @x13n can continue to be the ones to work on it, and we can list them as the owners of the subproject, but it'll fall under the purview of sig autoscaling.

Reply:
@DirectXMan12’s suggestion sounds good to me.

Reply:
@kubernetes/sig-network-misc the cluster-proportional-autoscaler doesn't seem to be listed under `sigs.yaml` (unless my searching is failing me).  I think it makes some amount of sense for it to be a subproject of sig-autoscaling (no change in ownership necessary, but it should fall under a sig umbrella), so I'd appreciate your opinions.

Reply:
@DirectXMan12 what you suggested sounds great. Also wanted to point out yet another component that might fall under this same umbrella: [cluster-proportional-vertical-autoscaler](https://github.com/kubernetes-incubator/cluster-proportional-vertical-autoscaler).

Reply:
cool, thanks for the heads up.  I'll update sigs.yaml with all of those

Reply:
I don't think this SIG owns an incubator project, unless it was created by SIG members who want it to be within the scope of the SIG...